experimental/generation/generator/packages/library/test/forms/unittests.form

```diff
@@ -91,10 +91,7 @@
             ]
         },
         "Percentage": {
-            "type": "string",
-            "$entities": [
-                "percentage"
-            ]
+            "type": "percentage"
```

Comment:
>percentage [](start = 21, length = 10)

This is not a valid JSON schema type.  This is why we had the schema, you should do $ref percentage.schema which has a type of number.